src/language_features/inlay_hints.rs

```diff
@@ -119,3 +124,60 @@ pub fn inlay_hints_response(
     );
     ctx.exec(meta, command)
 }
+
+#[derive(Debug)]
+pub struct InlayHintApplyParams {
+    pub selection_desc: String,
+}
+
+pub fn apply_inlay_hint(meta: EditorMeta, params: InlayHintApplyParams, ctx: &mut Context) {
+    let document = match ctx.documents.get(&meta.buffile) {
+        Some(doc) => doc,
+        None => return, // buffer is gone
```

Comment:
clippy should yell at you to use let-else here `let Some(document)  = ctx.docuemtns.get() else { return; };`